proxies/live/apiproxy/targets/ers-target.xml

```diff
@@ -113,14 +113,23 @@
                     <Condition>(request.header.nhsd-ers-referral-id ~~ ".+")</Condition>
                 </Step> <Step>
                     <Name>AssignMessage.Remove.x-request-id-header</Name>
-                </Step><Step>
-                    <Name>AssignMessage.Set.x-ers-amr-header</Name>
                 </Step><Step>
                     <Name>DecodeJWT-id-token</Name>
+                </Step><Step>
+                    <Name>AssignMessage.Set.x-ers-id-assurance-level-header</Name>
                 </Step><Step>
                     <Name>AssignMessage.Set.x-ers-acr-header</Name>
                 </Step><Step>
-                    <Name>AssignMessage.Set.x-ers-id-assurance-level-header</Name>
+                    <Name>AssignMessage.Set.x-ers-amr-header</Name>
+                </Step><Step>
+                    <Condition>(isFhirR4Path = true)</Condition>
+                    <Name>AssignMessage.SetOperationOutcomeFhirProfileVariableR4</Name>
+                </Step><Step>
+                    <Condition>(isFhirR4Path = false)</Condition>
+                    <Name>AssignMessage.SetOperationOutcomeFhirProfileVariableStu3</Name>
+                </Step><Step>
```

Comment:
This is where I wonder if completing this as part of a `FaultRule` would be a bit nicer as I guess these fields will just be unused if an fault isn't raised below?

Reply:
Done.